docs/src/components/page-parts/platform-integrations/QuasarIntegrations.vue

```diff
@@ -19,54 +19,84 @@
     <h2
       class="landing-heading landing-heading--medium q-mb-lg normal-line-height"
     >We have an impressive gear</h2>
-    <div class="integration-options row justify-center q-gutter-lg q-mb-xl">
+    <div class="row justify-center q-gutter-sm q-mb-xl">
       <q-card
         v-for="({path, label, icon, name}, cardIndex) in integrationOptions"
         :key="cardIndex"
-        class="card raise-on-hover column justify-center items-center cursor-pointer bg-brand-secondary"
+        class="card raise-on-hover column justify-center items-center cursor-pointer"
         flat
         @click="$router.push(`/${path}`)"
       >
         <q-icon
           :name="icon"
-          class="card__icon"
+          class="text-size-24 q-my-sm"
           color="brand-accent"
         />
-
-        <q-card-section class="text-white text-size-16 text-weight-bold letter-spacing-300">
+        <div class="text-size-14 text-weight-bold letter-spacing-263 text-brand-secondary">
           {{ label }}
-        </q-card-section>
-
-        <q-card-section class="text-white text-size-14 q-py-none q-px-xs letter-spacing-263 primary-line-height">
+        </div>
+        <div class="text-size-12 q-px-sm letter-spacing-100 text-brand-secondary">
           {{ name }}
-        </q-card-section>
+        </div>
       </q-card>
     </div>
+    <h2 class="landing-heading landing-heading--medium q-mb-lg normal-line-height">Discover Quasar Ecosystem</h2>
+    <template v-for="({about, options}, partIndex) in ecosystemParts" :key="`part-${partIndex}`">
+      <p class="text-size-16 letter-spacing-40 primary-line-height text-brand-secondary">{{ about }}</p>
+      <div class="row justify-center q-gutter-sm margin-bottom-36">
+        <router-link
```

Comment:
Use https://router.vuejs.org/api/#custom to use the card inside directly as a link item, so you can avoid `remove-a-tag-styles`

Reply:
Using `custom` prop really doesn't serve the purpose why we used `router-link` instead of `@click`. You won't still be able to click with the middle finger. Unless, the slot used therein is an `a` tag of course and we get back to the original problem. Unless I'm missing something about it?

Reply:
Mmm... Nope, you're correct, this won't solve the problem.
Create a wrapper managing both internal and external link (eg. `card-link` or whatever) as explained here then: https://router.vuejs.org/guide/advanced/extending-router-link.html

In that way you can apply `remove-a-tag-styles` directly into the wrapper